extra/1-currency-conversion.js

```diff
@@ -15,7 +17,11 @@ function convertToUSD() {}
   They have also decided that they should add a 1% fee to all foreign transactions, which means you only convert 99% of the £ to BRL.
 */
 
-function convertToBRL() {}
+function convertToBRL(price) {
+  let priceAfterFee = price * 0.99;
+  let priceInBRL = priceAfterFee * 5.7;
+  return Math.round(priceInBRL * 100) / 100;
+}
 
 /* ======= TESTS - DO NOT MODIFY ===== 
```

Comment:
Can we update this languages while we are here? https://codeyourfuture.slack.com/archives/C012UUW69S8/p1667338986259759

Instead of DO NOT MODIFY, maybe READ THESE CAREFULLY and ADD MORE if you find this useful?